src/eip4824.ts

```diff
@@ -82,6 +82,13 @@ router.get('/:space/proposals', async (req, res) => {
     type: 'proposal',
     id: proposal.id,
     name: proposal.title,
+    contentURI: {
+      body: proposal.body,
+      author: proposal.author,
+      ipfs: proposal.ipfs,
+      link: proposal.link
+    },
+    discussionURI: proposal.discussion,
```

Comment:
I get `"contentURI": {},` and no `discussionURI` field in response

Reply:
It should be fixed now, can you check?